test/unittests/t_util.cc

```diff
@@ -424,7 +425,8 @@ TEST_F(T_Util, MakePipe) {
   void *buffer_output = scalloc(100, sizeof(char));
   MakePipe(fd);
   write(fd[1], to_write.c_str(), to_write.length());
-  read(fd[0], buffer_output, to_write.length());
+  int bytes_read = read(fd[0], buffer_output, to_write.length());
+  EXPECT_EQ(bytes_read, to_write.length());
```

Comment:
Ah sorry, you mean in the comparison. Yes, string::length() returns an unsigned int. I'll fix that with a static_cast